src/cli/onefuzz/debug.py

```diff
@@ -337,11 +383,17 @@ def query(
             return raw_data
         return self._convert(raw_data)
 
+    def _query_parts(
+        self, parts: List[str], timespan: str, *, raw: bool = False
+    ) -> Any:
```

Comment:
Can we return `Union` instead of `Any`?

Reply:
Application Insights doesn't provide us any typing hints, so we'd have to cast it.  Given that the response is arbitrarily nested json, it wouldn't be very meaningful.

For the libfuzzer specific queries where we are projecting specific fields, I can see that being valuable.  I will investigate that as future work.